src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -83,29 +60,29 @@ export const BasicDateInput = (props) => {
       data-cy={`date-input-${id}`}
       direction="row"
       spacing={1}
-      alignItems="center"
       className={isDirty ? classes.dirtyInput : isDirty === false ? classes.notDirtyInput : ''}
+      sx={{ alignItems: 'center' }}
     >
-      <LocalizationProvider dateAdapter={AdapterDateFns}>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
         <DesktopDatePicker
           label={label}
           inputFormat={format}
-          minDate={minDate}
-          maxDate={maxDate}
-          renderInput={({ error: _, ...params }) => (
-            <TextField
-              id={`date-text-field-${id}`}
-              variant="outlined"
-              sx={{ flexGrow: 1 }}
-              size={size}
-              error={error?.message?.length > 0}
-              helperText={error?.message ?? ''}
-              {...params}
-            />
-          )}
+          minDate={dateProps.minDate ? dayjs(dateProps.minDate) : null}
+          maxDate={dateProps.minDate ? dayjs(dateProps.maxDate) : null}
+          timezone={TIME_ZONE}
+          slotProps={{
+            textField: {
+              id: `date-text-field-${id}`,
+              variant: 'outlined',
+              sx: { flexGrow: 1 },
+              size: 'small',
```

Comment:
Why the size is hard-coded but the corresponding prop still exists (lines 35 and 133) ? :thinking:

Reply:
A mess again... At the beginning, I set _size_ prop to 'small' trying to fix a display issue and I forgot to reset it at the end.